index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/8eb467c874.js" crossorigin="anonymous"></script>
+  </head>
+
+<body class="body">
+  <header class="header">
+    <div class="container"> 
+    
+     <a class="name" harf="index.html">Ibrahim Al-Yousefi</a>
```

Comment:
**[OPTIONAL]** - You are currently using the wrong property `harf`. I guess you mean to use `href`. Please try to fix this 🙏 🙏